packages/flutter_markdown/lib/flutter_markdown.dart

```diff
@@ -2,6 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+/// A library to render markdown formatted text.
```

Comment:
@vlidholt Could you add some more documentation to this comment (like how to use the library)? No urgent need to go and dart-doc the whole library but if we could get a few paragraphs of introductory material here that would go a long way for now.